mycore-iview2/src/main/java/org/mycore/iview2/iiif/MCRThumbnailImageImpl.java

```diff
@@ -66,15 +67,33 @@ protected MCRTileInfo createTileInfo(String id) throws MCRIIIFImageNotFoundExcep
                         .anyMatch(type::equals)) {
                         final String maindoc = derLink.getMainDoc();
                         if (maindoc != null) {
-                            final MCRTileInfo mcrTileInfo = new MCRTileInfo(derLink.getXLinkHref(), maindoc, null);
-                            final Optional<Path> tileFile = this.tileFileProvider.getTileFile(mcrTileInfo);
-                            if (tileFile.isPresent() && Files.exists(tileFile.get())) {
-                                return mcrTileInfo;
+                            Optional<MCRTileInfo> tileInfoForFile = createTileInfoForFile(derLink.getXLinkHref(),
+                                maindoc);
+                            if (tileInfoForFile.isPresent()) {
+                                return tileInfoForFile.get();
                             }
                         }
                     }
                 }
             }
+            Optional<String> linkedImage = MCRLinkTableManager.instance()
+                .getDestinationOf(mcrID, MCRLinkTableManager.ENTRY_TYPE_DERIVATE_LINK).stream().findFirst();
```

Comment:
The implementation iterates over all derivates and the first one which has a tiles will be used. 
It should work the same way with derivate_links and not only consider the first link.